frontend/src/components/waitlist/countryPicker.tsx

```diff
@@ -0,0 +1,62 @@
+import { useLocalization } from "@fluent/react";
+import { SelectHTMLAttributes, useEffect, useState } from "react";
+import { getLocale } from "../../functions/getLocale";
+
+type LocaleDisplayNames = Record<string, string>;
+type Territories = {
+  main: Record<
+    string,
+    {
+      localeDisplayNames: {
+        territories: LocaleDisplayNames;
+      };
+    }
+  >;
+};
+export const CountryPicker = (
+  props: SelectHTMLAttributes<HTMLSelectElement>
+) => {
+  const { l10n } = useLocalization();
+  const currentLocale = getLocale(l10n);
+  const [localeDisplayNames, setLocaleDisplayNames] =
+    useState<LocaleDisplayNames>();
+
+  useEffect(() => {
+    importTerritories(currentLocale).then((localeDisplayNames) => {
+      setLocaleDisplayNames(localeDisplayNames);
+    });
+  }, [currentLocale]);
+
+  const options = Object.entries(localeDisplayNames ?? {})
+    // cldr-localenames-modern also includes names of continents,
+    // whose territory codes consist of numbers.
+    // Since we're only interested in countries, filter those out:
+    .filter(
+      ([code]) =>
+        !["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"].some((nr) =>
+          code.includes(nr)
+        )
```

Comment:
It took a bit of digging, but now I understand the numbers comment! 

https://github.com/unicode-org/cldr-json/blob/main/cldr-json/cldr-localenames-modern/main/en/territories.json

Reply:
Good point, that could use less digging: 1c5a579d.